docs/migration-2-to-3.md

```diff
@@ -25,6 +25,37 @@ Summary of noticeable changes:
   directly in new packages.
 
 
+## Compile task
+
+Spring Boot 3 requires reflection information about parameters in compiled classes.
+If no such information is added, warning logs will complain about it.
+To avoid warnings, the `-parameters` flag must be added to the compile task.
+
+=== "Gradle"
```

Comment:
Do you mean `###`? The current version [does not render](https://github.com/SDA-SE/sda-spring-boot-commons/blob/ef2d93a5efdc6cbea4e03138943b69187b1d5517/docs/migration-2-to-3.md) nicely.

Reply:
Hmm, maybe it's time to publish the docs. That renders nicely with mkdocs as used in the public docs for mongodb-operator and in backstage. Should we prepare to publish GitHub pages or should I use another formatting?

<img width="860" alt="image" src="https://github.com/SDA-SE/sda-spring-boot-commons/assets/43337019/0f50ce63-5ad4-4710-8e52-087dccbd0308">

Reply:
see #343